pkg/http-metrics/pkg/db/metric.go

```diff
@@ -0,0 +1,28 @@
+package db
```

Comment:
Two concerns about this:
1) Is the data structure outdated? The actual one is:|
```
type RawResponseTime struct {
	Timestamp   time.Time
	Source      string
	Destination string
	Function    string
	Namespace   string
	Community   string
	Gpu         bool
	Latency     int
	StatusCode  int
	Description string
	Path        string
	Method      string
}
```
2) to avoid this inconsistency, can we import the data structure directly from the edge autoscaler repository? Or at least we can add a TODO to extract this data structure into a dedicated package shared by both repositories

Reply:
The struct is different since some of the information is not needed. In order to not overload the database, the minimum amount of information has been stored.